shared/js/ui/templates/shared/rating-hero.es6.js

```diff
@@ -69,8 +71,15 @@ function siteRatingSubtitle (isCalculating, rating, isWhitelisted) {
 
     // deal with other states
     let msg = 'Privacy Grade'
-    // site is whitelisted
-    if (!isActive) {
+    // tracker blocking opt in experiment
+    if (blockingDisabled) {
```

Comment:
When tracker blocking is disabled, show `Privacy Grade` in the popup scorecard section when page doesn't have any trackers, or `Trackers found` when it does.